### DISCUSSION TITLE
MinTLS1.3 too restrictive

### DISCUSSION THREAD

**Discussion by xoxys** [bpg/terraform-provider-proxmox#920] [Q&A] [answered]
The PR https://github.com/bpg/terraform-provider-proxmox/pull/596 has set the required min TLS version to TLS1.3. In my case, I have a Nginx server running on CentOS7 in front of my Proxmox server as a reverse proxy, which doesn't support TLS1.3.

As TLS1.2 is still considered as secure, would it be possible to change the min required TLS version to 1.2 or make it at least configurable? Would be happy to provide the PR.

**Comment by bpg-dev** [comment, marked as answer]
Hi @xoxys 👋🏼 

Yeah, there was another report about TLS1.3 in #657. I think it make sense to add `min_tls` argument to the provider's [arguments](https://registry.terraform.io/providers/bpg/proxmox/latest/docs#argument-reference) for setting up the min TLS version, with keeping 1.3 as the default.

A PR would be awesome! ❤️
